packages/fast-components/src/switch/switch.ts

```diff
@@ -0,0 +1,100 @@
+import { attr, observable } from "@microsoft/fast-element";
+import { FormAssociated } from "../form-associated";
+import { keyCodeSpace } from "@microsoft/fast-web-utilities";
+import { bool } from "../utilities";
+
+export class Switch extends FormAssociated<HTMLInputElement> {
```

Comment:
Is this a form associated element? Do we want to base our switch off a checkbox?